client/app/components/OnboardingGuide/GuideCoverPhotoPage.js

```diff
@@ -46,17 +47,13 @@ const GuideCoverPhotoPage = (props) => {
       ]),
     ]),
 
-    a({ className: css.nextButton, href: pageData.cta }, t('web.admin.onboarding.guide.cover_photo.add_your_own')),
+    a({ className: css.nextButton, href: Routes.admin_look_and_feel_edit_path() }, t('web.admin.onboarding.guide.cover_photo.add_your_own')),
```

Comment:
To clarify further, there's never a need for routes to go into redux store. They are static from the js app perspective so there's no need to trigger refresh as a result of routes changing.